### ISSUE TITLE
Sort by dataset size

### ISSUE THREAD

**Issue by naught101** [TerriaJS/magda#174]
There are a lot of datasets on data.gov.au, and many of them are tiny, and not all that interesting. It would be nice to be able to sort/filter datasets by filesize, and possibly also by sample size/dimensionality.

**Comment by tkeuneman** [comment]
@naught101 What would filtering by filesize do? Are you looking for a particular filesize in particular, and why?

**Comment by naught101** [comment]
@tkeuneman Yep, being able to filter out datasets by filesize would be a great solution.

Basically, when I was looking through the datasets for something to analyses for practice, the results list kept coming up with really interesting looking datasets that, when I downloaded them, turned out to have only 5-10 rows of data, which is pretty useless for most purposes. I eventually gave up looking for something interesting, because the majority of what I looked at was too small to bother with. Being able to filter or sort would allow me to find stuff that's interesting AND big enough to do some real analysis with.

**Comment by tkeuneman** [comment]
@naught101 Thanks for the clarification. 

Do you think the search filters at search.data.gov.au helps with this? 

You can also preview datasets. It's a little bit hidden at the moment but we're making them more prominent. You can see it by clicking on 'show preview' on a dataset like this: http://search.data.gov.au/dataset/ds-dga-ab7eddce-84df-4098-bc8f-500d0d9776d1/details

**Comment by naught101** [comment]
I can't see how the search filters there would help with this problem.

The preview feature does kind of help with the problem, but in a pretty slow way - I have to click the dataset, wait for the details page to load, then preview each file I'm interested in, and I have to do that for any given dataset. 

It would be much nicer with a filter for file size or sample size.

**Comment by naught101** [comment]
Also, previews don't seem to be working for some files (some TSV files I'm looking at)

**Comment by tkeuneman** [comment]
We're still working on the previews as well as some preliminary data visualisation, which should help with seeing if you can do anything interesting with the data. 

We’re not in a position to work on this right away but we’ll keep it on the roadmap in the new financial year. 

Thanks so much for the feedback, keep it coming!
